modules/liftover/2.0/src/convert_for_liftover.py

```diff
@@ -22,57 +22,90 @@ def main():
     # read required second argument from shell command as path to resulting .bed file
```

Comment:
I should comment here that Kostia reviewed this on my last PR that was already merged into master and I accidentally overwrote it with the old version while completing the version bump process.